Resources/Prototypes/DeltaV/Catalog/Fills/Boxes/general.yml

```
@@ -16,3 +16,22 @@
     whitelist:
       components:
       - EncryptionKey
+      
+- type: entity
+  name: justice encryption key box
+  parent: BoxCardboard
```

```suggestion
  parent: BoxEncryptionKeyPassenger
```
In hindsight, I should've double checked the Prison Encryption Key box to make sure it doesn't use redundant components; but it does, and I assume you've copied the Prison Box so I'll suggest the right thing to do here and eventually fix the Prison Key Box too.